packages/motion/src/helpers.js

```diff
@@ -85,7 +85,7 @@ export function getWebpackConfig(state: State, config: Motion$Config, cli: CLI,
         Path.join(Path.normalize(Path.join(__dirname, '..')), 'node_modules'),
         Path.join(Path.normalize(Path.join(__dirname, '..', '..')), 'node_modules')
       ],
-      packageMains: ['webpack', 'browser', 'web', 'browserify', 'jsnext:main', 'main']
+      packageMains: ['webpack', 'browser', 'web', 'browserify', 'main', 'jsnext:main']
```

Comment:
Note: we should probably move this back up and test it with react-router. We'd need to support `jsnext:main`(perhaps pundle is up to this?). I'm not familiar with how it works but I assume it should direct us on how to compile it.